pkg/kv/params/database.go

```diff
@@ -10,12 +10,27 @@ type KV struct {
 	DynamoDB *DynamoDB
 }
 
+// EnableMetrics returns a copy of params with Metrics enabled for Postgres.
+// This is used only when we like to get parameters for the main run of lakeFS
+// as the is only one metrics set we expose
+func (p KV) EnableMetrics() KV {
+	if p.Postgres == nil || p.Postgres.Metrics {
+		return p
+	}
+	// make a copy of postgres settings and set metrics on
+	pg := *p.Postgres
+	pg.Metrics = true
+	p.Postgres = &pg
+	return p
+}
+
 type Postgres struct {
 	ConnectionString      string
 	MaxOpenConnections    int32
 	MaxIdleConnections    int32
 	ConnectionMaxLifetime time.Duration
 	ScanPageSize          int32
+	Metrics               bool
```

Comment:
Add the new param to the configuration doc

Reply:
Currently is it not configurable - having it as part of the postgres configuration is to enable us to control on which pgxpool we like to have the metrics enabled.